build.sbt

```diff
@@ -210,8 +210,9 @@ def dockerSettings: Seq[Setting[_]] = Seq(
   dockerUpdateLatest := true,
   dockerRepository := sys.props.get("docker.registry"),
   dockerUsername := sys.props.get("docker.username").orElse(Some("cloudstateio")).filter(_ != ""),
-  dockerBaseImage := "openjdk:8-jre-slim-buster",
-//  dockerBuildOptions += "--no-cache",
+  dockerBaseImage := DockerBaseImageVersion,
+  // when using tags like latest, uncomment below line, so that local cache will not be used.
+  //  dockerBuildOptions += "--no-cache",
```

Comment:
@srikanthops If one always wants to use --no-cache with latest-deps, couldn't we add it automatically?